src/components/Disclosure/Disclosure.scss

```diff
@@ -24,11 +24,14 @@ $gap: 8px;
         font-size: inherit;
         line-height: inherit;
         font-weight: inherit;
+        border-radius: 2px;
```

Comment:
This should not be a fixed value. And I doubt focus should have a radius at all by default. It component that determines the radius

Reply:
Introduced --g-focus-border-radius